src/System.IO.FileSystem.AccessControl/dir.props

```diff
@@ -3,5 +3,7 @@
   <Import Project="..\dir.props" />
   <PropertyGroup>
     <AssemblyVersion>4.0.2.0</AssemblyVersion>
+    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
@weshaggard We need the functionality for sure. We could copy the code I suppose but I'm a little worried about missing fixes given this is code we haven't really extensively utilized.